components/butterfly/tests/common/mod.rs

```diff
@@ -60,16 +60,13 @@ pub fn start_server(name: &str, ring_key: Option<SymKey>, suitability: u64) -> S
                                  Some(String::from(name)),
                                  None,
                                  Box::new(NSuitability(suitability))).unwrap();
-    server.start_rsw_mlw(&Timing::default())
+    server.start_rsw_mlw_smw(&Timing::default())
           .expect("Cannot start server");
     server
 }
 
 pub fn member_from_server(server: &Server) -> Member {
```

Comment:
Should this be `member_from_server_smr` instead?

Reply:
I don't think so?  `Server::start_rsw_mlw_smw` is so named because it takes a write lock on `self.member`.  I don't see any read locks in that function.

Reply:
I think @christophermaier is saying `member_from_server` should have an `_smr` annotation because of [the `read` that happens here](https://github.com/habitat-sh/habitat/blob/e14726f30c06b03d7ceeee17dfc42d545e89bee6/components/butterfly/tests/common/mod.rs#L70).

Reply:
Ah yes, apologies for missing the meaning there.